src/components/3d-hover-effect-glow/3d-hover-effect-glow.scss

```diff
@@ -0,0 +1,21 @@
+:host(limel-3d-hover-effect-glow) {
+    transition: background, opacity;
+    transition-timing-function: ease-out; // Must be the same as the 3D Element itself
+    transition-duration: 0.8s; // Must be the same as the 3D Element itself
+
+    display: block;
+    pointer-events: none;
```

Comment:
Ah, yeah, of course… this means that `:hover` will never be applied to this element… such a _tiny_ little annoyance that the opacity change on `:hover` can't be styled in here… 😑

But this is still much better than having to add both a div and a mixin 👍